open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconChooserPanel.java

```diff
@@ -606,6 +614,24 @@ private void displayIconRecords(final List<IconRecord> recList, boolean canCance
         myLoader = new Thread(new BuildIconGridWorker(recList, canCancel));
         myLoader.start();
     }
+    
+    /**
+     * Shows resizes icons and displays loading screen until done.
+     */
+	public void IconResizer(List<IconRecord> recList,int tileWidth,boolean canCancel) 

```

Comment:
Methods other than the constructor should start with a lowercase letter, and generally be verby rather than nouny.  So maybe something like 'resizeIcons'.